### DISCUSSION TITLE
Fixed: Bugs of update 202210131

### DISCUSSION THREAD

**Discussion by rern** [rern/rAudio#1068] [General]
Hi guys,

Sorry for any inconveniences caused by update 202210131.
Please update again.

**Comment by Raitsa** [comment]
Thanks for taking very good care of us all, hugely appreciated 😊

**Comment by ceejayemm** [comment]
@rern 
Didn't notice any problems with 202210131 but updated to this version  to stay on top of thinks.

Thanks

Chris
